plugin/federation/federation.go

```diff
@@ -382,7 +382,7 @@ func (f *federation) GenerateCode(data *codegen.Data) error {
 				return err
 			}
 
-			requiresFile := data.Config.Resolver.Dir() + "/federation.requires.go"
+			requiresFile := data.Config.Federation.Dir() + "/federation.requires.go"
```

Comment:
@ldebruijn I've reverted this from your proposed change as it breaks the implementation and the test code generation.